README.md

```diff
@@ -71,8 +71,7 @@ CONNECTOR_LEDGERS={
     "currency": "EUR",
     "plugin": "ilp-plugin-payment-channel-framework",
     "options": {
-      "server": "btp+wss://wallet1.example:1234/example_path",
-      "token": "shared_secret"
+      "btpUri": "btp+wss://username:shared_secret@wallet1.example:1234/example_path"
```

Comment:
The reasoning to use `server` + `secret` here was that it is easier to understand next to the other plugins like XRP escrow (cc. @justmoon)

Reply:
see [my comment above](https://github.com/interledgerjs/ilp-plugin-payment-channel-framework/pull/27#discussion_r140419857)